doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Prediction\n",
+    "\n",
+    "```{tip}\n",
+    "We strongly recommend using [Ray Datasets](https://docs.ray.io/en/latest/data/examples/nyc_taxi_basic_processing.html#parallel-batch-inference) or {doc}`AIR predictors </ray-air/predictors>` to develop batch prediction, which will enable you to build it faster and more easily, and get the built-in benefits like auto-scaling actor pool. If you think your use case cannot be supported by Ray Datasets or AIR, please file an issue in [Ray GitHub](https://github.com/ray-project/ray/issues) to get advice before you proceed. \n",
```

Comment:
could you try:

```suggestion
    "We strongly recommend using [Ray Datasets](https://docs.ray.io/en/latest/data/examples/nyc_taxi_basic_processing.html#parallel-batch-inference) or {doc}`AIR predictors <air-predictors>` to develop batch prediction, which will enable you to build it faster and more easily, and get the built-in benefits like auto-scaling actor pool. If you think your use case cannot be supported by Ray Datasets or AIR, please file an issue in [Ray GitHub](https://github.com/ray-project/ray/issues) to get advice before you proceed. \n",
```

to see if this compiles properly?

Reply:
Thanks for suggestion, "{doc}`AIR predictors </ray-air/predictors>`" actually works fine.

Reply:
the reason why you should do the above is so that the link does not break when the section moves (when we do documentation refactors)

Reply:
What if the "air-predictors" ref gets changed as well? We did have such change as part of renaming.